2010-01-05  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/et_EE: Change first_weekday and first_workday to 2.

---
 localedata/locales/et_EE |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/localedata/locales/et_EE
+++ b/localedata/locales/et_EE
@@ -178,8 +178,8 @@
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 week    7;19971130;4
-first_weekday 1
-first_workday 1
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_MESSAGES
